docs/set-up.asciidoc

```diff
@@ -39,32 +40,278 @@ include::./custom-stack.asciidoc[]
 
 include::./lambda.asciidoc[]
 
+include::./typescript.asciidoc[]
```

Comment:
What do you think of swapping the order of these includes so that all "Get started with..." pages are grouped together? 

<img width="353" alt="Screen Shot 2022-07-28 at 2 30 07 PM" src="https://user-images.githubusercontent.com/5618806/181641045-22cf8a1d-3f12-4695-908f-0128a5abdfe6.png">

Reply:
Yup, I'm fine with that. I'll move it up.

A couple notes on this.

1. The "Monitoring AWS Lambda ..." entry is named that in the few APM agents that added Lambda support. E.g. https://www.elastic.co/guide/en/apm/agent/python/current/index.html   I guess that page title has better Google juice than "Get started with AWS Lambda". Dunno. It *is* basically still a "getting started" page.

2. I had originally started making larger TOC changes here, but moved that to separate work (if I get to it). I was thinking a TOC something like this:

```
TOC                     # add brief intro prose to this page
(Getting started)
Introduction
Get started    # fastest 1-2-3 get started, with links to tech-specific quick start pages
    Get started with Express
    Get started with hapi
    Get started with Koa
    Get started with Fastify
    Get started with Restify
    *Get started with TypeScript    # ???
    *Get started with ...           # ??? Next.js, etc.
    ~Get started with a custom Node.js stack~  # maybe drop this
Supported technologies
Features        # new page that gives a one pager with a short pitch for defining features, inspiration https://tailscale.com/kb/1169/features/, I'm not married to doing this

(Reference)
Starting the agent          # maybe incorporate the "ES Modules support" (incl. TypeScript) docs in this page? Isn't quite right. Hrm.
Configuration               # merge "Configuration", "Configuring the agent", and "Configuration options" into one page here. *Consider* separate subpages for related-groups of config (a la Java), though I *really* like having one searchable doc page for all config. Perhaps just reconsider the TOC organization of the page and still do in a single page.
Tracing APIs                # I like some of the extra content here from Java docs
    Agent API
    Transaction API
    Span API
    OpenTelemetry Bridge
    OpenTracing Bridge

(Feature References, Howtos, etc.)    XXX perhaps re-order these.
Metrics
Log correlation
Source map support
Distributed tracing
Message queues
Manual instrumentation      # Merge "custom transactions" and "Custom spans" into this

(Extra stuff)
Performance Tuning
Troubleshooting
Upgrading
    ...
Release notes
    ...
```

Reply:
Moved up in commit c9fb5c34